stock_manual_transfer/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License: LGPL-3
+
+Functional Rationale:
+---------------------
+
+It allows you to make a stock transfer manually using a custom route.
```

Comment:
This module allows you to trigger transfers using a specific route, as it were triggered by a reordering rule.